tests/h/streamer/filter_test.py

```diff
@@ -27,20 +27,21 @@ class TestFilterHandler:
             (["http://example.com"], "https://example.com/?", True),
         ],
     )
-    def test_it_matches_uri(self, factories, query_uris, ann_uri, should_match):
+    def test_it_matches_uri(self, make_annotation, query_uris, ann_uri, should_match):
+        ann = make_annotation(target_uri=ann_uri)
+
         query = {
             "match_policy": "include_any",
             "actions": {},
             "clauses": [{"field": "/uri", "operator": "one_of", "value": query_uris}],
         }
         handler = FilterHandler(query)
 
-        ann = factories.Annotation(target_uri=ann_uri)
         assert handler.match(ann) is should_match
 
-    def test_it_matches_id(self, factories):
-        ann_matching = factories.Annotation(target_uri="https://example.com")
-        ann_non_matching = factories.Annotation(target_uri="https://example.net")
+    def test_it_matches_id(self, make_annotation):
+        ann_matching = make_annotation()
+        ann_non_matching = make_annotation()
```

Comment:
The `target_uri` wasn't used in the test and a random one should be provided by the fixture.